apps/docs/content/components/CTAForm.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: CTAForm
+status: Experimental
+source: https://github.com/primer/brand/tree/main/packages/react/src/CTAForm/CTAForm.tsx
+figma: 'https://www.figma.com/file/BJ95AjraesmRCWsKA013GS/Primer-Brand?node-id=1377%3A30754'
+storybook: '/brand/storybook/?path=/story/components-CTAForm--playground'
+description: Use CTAForm to create a form that is used to collect minimal information from users.
+---
+
+import {
+  CTAForm,
+  InlineLink,
+  FormControl,
+  TextInput,
+  Checkbox,
+  Text,
+} from '@primer/react-brand'
+
+```js
+import {CTAForm} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+```jsx live
+<CTAForm>
+  <CTAForm.Input>
+    <FormControl required>
+      <FormControl.Label>Your work email address</FormControl.Label>
+      <TextInput placeholder="name" />
+    </FormControl>
+  </CTAForm.Input>
+  <CTAForm.Confirm>
+    <FormControl required>
+      <FormControl.Label>
+        <Text size="300" variant="muted">
+          I agree to the{' '}
+          <InlineLink href="www.github.com">Privacy Policy</InlineLink> and{' '}
+          <InlineLink href="www.github.com">Terms of Service</InlineLink>
+        </Text>
+      </FormControl.Label>
+      <Checkbox name="confirm" />
+    </FormControl>
+  </CTAForm.Confirm>
+  <CTAForm.Action>Subscribe</CTAForm.Action>
+</CTAForm>
+```
+
+### Component without confirm block
+
+```jsx live
+<CTAForm>
+  <CTAForm.Input>
+    <FormControl required>
+      <FormControl.Label>Your work email address</FormControl.Label>
+      <TextInput placeholder="name" />
+    </FormControl>
+  </CTAForm.Input>
+  <CTAForm.Action>Subscribe</CTAForm.Action>
+</CTAForm>
+```
+
+### Using onChange & onSubmit handlers
+
+```jsx live
+<CTAForm onSubmit={() => alert('onSubmit event triggered')}>
+  <CTAForm.Input onChange={() => alert('onChange event triggered')}>
```

Comment:
I think using a console.log or on-screen counter would be more ideal here, as it's currently firing the alert on every key press and breaking momentum of the user input. The alert would perhaps be more suitable for the submit handler only?

https://user-images.githubusercontent.com/13340707/219355159-e6b43c77-4fb5-4e8d-a2f8-8b5df7287465.mov